source/scripting_v3/GTA/Camera/GameplayCamera.cs

```diff
@@ -444,6 +444,53 @@ public static float FirstPersonAimCameraZoomFactor
 		/// </value>
 		public static bool IsFirstPersonAimCamActive => Function.Call<bool>(Hash.IS_FIRST_PERSON_AIM_CAM_ACTIVE);
 
+		/// <summary>
+		/// Gets a value indicating whether a first person vehicle camera is active.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is a first person vehicle camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsFirstPersonVehicleCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_FIRST_PERSON_VEHICLE_INTERIOR_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether any cinematic camera is active.
+		/// Note that this will also return true if in first person view while inside a vehicle.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is a cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsAnyCinematicCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether an idle cinematic camera is active.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is an idle cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsIdleCinematicCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_IDLE_CAM_RENDERING);
```

Comment:
It would be better if these properties are named like `IsAnyCamRendering` and `IsIdleCamRendering` (on the condition that these props moves to the new static class), not like `IsAnyCamActive`. Test `IS_FOLLOW_PED_CAM_ACTIVE` or `IS_FOLLOW_VEHICLE_CAM_ACTIVE` with a scripted camera rendering to see the difference. R* seem to distinguish the term `active` or `rendering` for camera natives in consistent ways.

They seem to use the term `active` when natives detect if the director they are insterested in uses its variables and/or its `camBaseCamera` instance that meets the condition. No need to the director is dominant which is required to have its `camBaseCamera` instance render. While they use the term `rendering` when natives detect if the director is dominant so its camera can be rendered. Of course some natives checks additional conditions (Counterexample: `IS_CINEMATIC_CAM_RENDERING` only checks the dominant one is the same as the cinematic cam director).